wordpress.yaml

```diff
@@ -1,7 +1,149 @@
+                    openssl dhparam -out /etc/nginx/dhparam.pem 2048
+                fi
+        
+    - name: Copy Conf. Snippets pointing SSL key and certificate
+      copy:
+        src: "/vagrant/self-signed.conf"
+        dest: "/etc/nginx/snippets/"
+        remote_src: yes
+        
+    - name: Copy ssl-params
+      copy:
+        src: "/vagrant/ssl-params.conf"
+        dest: "/etc/nginx/snippets/"
+        remote_src: yes
+        
+    - name: Copy Nginx config       
+      template:
+        src: "/vagrant/nginx.conf.j2"
+        dest: "/etc/nginx/sites-available/{{ wp_root_dir }}"
+        
+    - name: Symbolic link on Wordpress nginx configuration file      
+      file:
+        src: "/etc/nginx/sites-available/{{ wp_root_dir }}"
+        dest: "/etc/nginx/sites-enabled/{{ wp_root_dir }}"
+        state: link
+        
+    - name: Unlink Default nginx config      
+      file:
+        path: /etc/nginx/sites-enabled/default
+        state: absent
+        
+    - name: Create WP Database
+      mysql_db:
+        name: "{{ db_name }}"
+        state: present
+        login_user: root
+        login_password: "{{ mysql_password }}"
+    
+    - name: Create WordPress Database user
+      mysql_user:
+        name: "{{ db_user }}"
+        password: "{{ db_password }}"
+        priv: "wpDB.*:ALL,GRANT"
+        state: present
+        login_user: root
+        login_password: "{{ mysql_password }}"
+        
+    - name: Extract Wordpress file        
+      unarchive:
+        src: "{{ url_wordpress }}"
+        dest: "/var/www/"
+        remote_src: yes
+        
+    - name: Copy wordpress config php file (wp database connection)        
+      template:
+        src: "/vagrant/wp-config.php.j2"
+        dest: "/var/www/{{ wp_root_dir }}/wp-config.php"
+        force: yes
+    
+    - name: Setup owner and group privileges to www-data and 755 permissions.       
+      file:
+        path: /var/www/{{ wp_root_dir }}
+        state: directory
+        recurse: yes
+        owner: www-data
+        group: www-data
+        mode: 0755
+    
+    - name: Setup 777 permissions to wp-content, so upload works
+      file:
+        path: /var/www/{{ wp_root_dir }}
+        state: directory
+        recurse: yes
+        owner: www-data
+        group: www-data
+        mode: 0777
+            
+    - name: Download WP-Cli file      
+      get_url:
+        url: "{{ url_wpcli }}"
+        dest: /usr/local/bin/wp
+        mode: 0755
+    
+    - name: Install WP-CLI     
+      shell: 'sudo -u ber -i -- wp core install --url="{{ wp_iphost }}" --title="{{ title }}" --admin_user="{{ wp_admin_user }}" --admin_email="{{ wp_admin_mail }}" --admin_password="{{ wp_admin_password }}" --path="/var/www/{{ wp_root_dir }}"'
+      
+    - name: Install Theme     
+      shell: 'sudo -u ber -i -- wp theme install twentynineteen --activate --path="/var/www/{{ wp_root_dir }}"'
```

Comment:
in ansible there is a proper option to run command as user. Also it's not a good habbit to run same command every time. I suppose you want to run it once when there is no wordpress running